.changeset/spotty-stingrays-camp.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-admin": patch
+---
+
+Update the default icon in `ContentScopeSelect` from `<Domain />` to `<Language />`
```

Comment:
It could be argued that this is a breaking change -> do we want to change that in main?